subsys/bluetooth/host/conn.c

```diff
@@ -2043,6 +2043,7 @@ struct bt_conn *bt_conn_create_slave_le(const bt_addr_le_t *peer,
 			if (err && (err != -EALREADY)) {
 				BT_WARN("Directed advertising could not be"
 					" started: %d", err);
+				bt_conn_unref(conn);
```

Comment:
Even though these are small, one-line fixes, they're really independent issues and should therefore be in independent commits (can still be the same PR though), so please split these up. Note that I've added the 1.14 backport tag since I think these fixes are important enough to go to the LTS branch as well.

Reply:
Commit split